pattr.py

```diff
@@ -106,6 +106,8 @@ def nick_passes(nickname):
         return False
     elif len(nickname) == 0:
         return False
+    elif nickname is 'pattrbot':
```

Comment:
Would you mind making this
```
elif nickname == 'pattrbot':
```
to make it consistent with the rest of our statements?